features/search/screens/ProfileSearch.tsx

```diff
@@ -0,0 +1,133 @@
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { useCallback } from "react";
+import {
+  FlatList,
+  Keyboard,
+  Platform,
+  TextStyle,
+  View,
+  ViewStyle,
+} from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+import { Text } from "@design-system/Text";
+import { translate } from "@i18n";
+import { ProfileSearchItem } from "../components/ProfileSearchItem";
+import { ProfileSocials } from "@data/store/profilesStore";
+import { useAppTheme, ThemedStyle } from "@theme/useAppTheme";
+
+type ProfileSearchProps = {
+  navigation: NativeStackNavigationProp<any>;
+  profiles: { [address: string]: ProfileSocials };
+  groupMode?: boolean;
+  addToGroup?: (member: ProfileSocials & { address: string }) => void;
+};
+
+export default function ProfileSearch({
+  navigation,
+  profiles,
+  groupMode,
+  addToGroup,
+}: ProfileSearchProps) {
+  const insets = useSafeAreaInsets();
+  const { theme, themed } = useAppTheme();
+
+  const keyExtractor = useCallback((address: string) => address, []);
+
+  const renderItem = useCallback(
+    ({ item }: { item: string }) => (
+      <ProfileSearchItem
+        address={item}
+        socials={profiles[item]}
+        navigation={navigation}
+        groupMode={groupMode}
+        addToGroup={addToGroup}
+      />
+    ),
+    [profiles, navigation, groupMode, addToGroup]
+  );
+
+  const renderHeader = useCallback(
+    () => (
+      <View style={themed($sectionTitleContainer)}>
+        <Text preset="formLabel" style={themed($sectionTitleSpacing)}>
+          {translate("search_results")}
+        </Text>
+      </View>
+    ),
+    [themed]
+  );
+
+  const renderFooter = useCallback(
+    () => (
+      <View style={[themed($footer), { marginBottom: insets.bottom + 55 }]}>
```

Comment:
Where is 55 coming from?

Reply:
@alexrisch Historically, to leave some room at the bottom of the screen when no result. Happy to remove the hardcoded value, should we?